compatibility/src/rand.rs

```diff
@@ -0,0 +1,24 @@
+// Simplified case, taken from:
+// https://github.com/rust-random/rand/blob/19404d68764ed08513131f82157e2ccad69dcf83/rand_pcg/src/pcg64.rs#L37-L40
```

Comment:
This is _probably_ simple enough to not need to mention the rand license, but do that just in case.

Reply:
Added a comment mentioning that their license is `MIT OR Apache-2.0`